build/docker/alpine.Dockerfile

```
@@ -51,5 +51,27 @@ RUN apk --no-cache --update add \
 
 RUN dotnet --version
 
+RUN echo 'https://dl-cdn.alpinelinux.org/alpine/v3.14/community' >> /etc/apk/repositories
+RUN echo 'https://dl-cdn.alpinelinux.org/alpine/v3.14/main' >> /etc/apk/repositories
+RUN apk --update --no-cache add \
```

```suggestion
RUN apk --no-cache add \
```

don't update packages